crates/sui-benchmark/src/lib.rs

```diff
@@ -135,8 +136,9 @@ pub trait ValidatorProxy {
 
 // TODO: Eventually remove this proxy because we shouldn't rely on validators to read objects.
 pub struct LocalValidatorAggregatorProxy {
-    _qd_handler: QuorumDriverHandler<NetworkAuthorityClient>,
-    qd: Arc<QuorumDriver<NetworkAuthorityClient>>,
+    _qd_handler: QuorumDriverHandler<NetworkAuthorityClient, IgnoreSignatureVerifier>,
```

Comment:
Do we still need this?

Reply:
I am actually not sure :) It does have some receiver so dropping it is not a no-op, so I did not want to change this line here